src/styles/main.css

```
@@ -0,0 +1,236 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+main {
+  margin: 64px 0;
+  min-height: 65vh;
+}
+
+.main-title {
+  font-family: Poppins;
```

Stylelint (17.5.0)

[error] Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

(font-family-no-missing-generic-family-keyword)